x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning.tsx

```diff
@@ -33,13 +48,26 @@ export const RelevanceTuning: React.FC<Props> = ({ engineBreadcrumb }) => {
           <EuiTitle size="l">
             <h1>{RELEVANCE_TUNING_TITLE}</h1>
           </EuiTitle>
+          <EuiText>
+            <EuiTextColor color="subdued">
+              {i18n.translate(
+                'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.description',
+                {
+                  defaultMessage: 'Set field weights and boosts',
+                }
+              )}
+            </EuiTextColor>
+          </EuiText>
         </EuiPageHeaderSection>
       </EuiPageHeader>
-      <EuiPageContent>
-        <EuiPageContentBody>
-          <FlashMessages />
-        </EuiPageContentBody>
-      </EuiPageContent>
+      <EuiSpacer />
+      <FlashMessages />
```

Comment:
Just to check - IIRC there should be some custom callouts here over either invalid schema or new/unconfirmed schema. Are those getting added in a separate PR? If so, do we want to add a quick TODO note?

Reply:
Ohhhhh right. I do have this, which I think may cover it. https://github.com/JasonStoltz/kibana/blob/relevance-tuning-4/x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_form.tsx#L45